cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -40,16 +40,18 @@ if(OMR_ENV_DATA64)
 		)
 	endif()
 else()
-	list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
-		-m32
-		-msse2
-	)
-	list(APPEND OMR_PLATFORM_EXE_LINKER_OPTIONS
-		-m32
-	)
-	list(APPEND OMR_PLATFORM_SHARED_LINKER_OPTIONS
-		-m32
-	)
+	if (NOT (CMAKE_SYSTEM_PROCESSOR MATCHES "ppc"))
```

Comment:
we should be testing against our "normalized" variables. IE `if(NOT OMR_ARCH_POWER)`